docs/index.rst

```diff
@@ -298,6 +298,23 @@ See `Syncing Ska data
 for details on automatically syncing these files for a standalone linux or Mac
 environment.  For Matlab tools on Windows a separate mechanism will be provided.
 
+Environment Variables
+---------------------
+
+The following environment variables are used by proseco:
+
+- ``AGASC_HDF5_FILE``: path to AGASC HDF5 file for getting AGASC star data. This
+  overrides the default value of ``$SKA/data/agasc/proseco_agasc_<latest>.h5``.
+- ``AGASC_SUPPLEMENT_ENABLED``: set to ``"False"`` to disable using the AGASC
+   supplement. This is for testing and should not be used in production.
+- ``PROSECO_IGNORE_MAXAGS_CONSTRAINTS``: if set then do not update ``maxmag`` in the
+  catalog to prevent search hits clipping.
+- ``PROSECO_OR_IMAGE_SIZE``: override the default OR image size of 8x8. Can be one of
+  "4", "6", or "8".
+- ``PROSECO_PRINT_OBC_CAT``: if set then create and print a debug catalog while doing
+  catalog merging.
+- ``SKA``: root directory for Ska3 runtime environment
```

Comment:
I think from proseco SKA is only used for defining SKA/data but I haven't re-checked that.